dataprofiler/reports/graphs.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+import math
+import warnings
+
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+
+def plot_histograms(profiler, columns=None):
+    """
+    Take a input of StructuredProfiler class and a list of specified column
+    names and then plots the histograms of those that are int or float
+    columns.
+
+    :param profiler: StructuredProfiler variable
+    :type profiler: StructuredProfiler
+    :param columns: list of column names to be plotted
+    :type columns: list
+    :return: returns fig
+    :rtype: fig
+    """
+
+    # get all inds to graph, raise error if user specified doesn't exist (part 1)
+    inds_to_graph = []
+    if not columns:
+        inds_to_graph = list(range(len(profiler.profile)))
+    else:
+        for column in columns:
+            col = column
+            if isinstance(col, str):
+                col = col.lower()
+            if col not in profiler._col_name_to_idx:
+                raise ValueError("Column \"" + str(col) + "\" is not found as a "
+                                                       "profiler column")
+            inds_to_graph.extend(profiler._col_name_to_idx[col])
+        sorted(inds_to_graph)
+    # get all columns which are either int or float (part2)
+    def is_index_graphable_column(ind_to_graph):
+        """
+        This function filters ind_to_graph by returning false if there is a
+        data type that is not a int or float, otherwise true
+        """
+        col_profiler = profiler.profile[ind_to_graph]
+        data_compiler = col_profiler.profiles['data_type_profile']
+        if data_compiler.selected_data_type not in ['int', 'float']:
+            return False
+        return True
```

Comment:
You could simplify this to just `return data_compiler.selected_dataype in ['int', 'float']`, not crucial, but would be cleaner

Reply:
hmm with that I am getting:

AttributeError: 'ColumnPrimitiveTypeProfileCompiler' object has no attribute 'selected_dataype'

Reply:
Stay with the conditions because others maybe added